### PR TITLE
Change nginx to be statically linked

### PR DESCRIPTION
**Pull request by dkubb**

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review.

File: alpine-rails-nginx/Dockerfile [hunk 1]
```diff
@@ -4,8 +4,8 @@ FROM alpine:edge
 MAINTAINER Dan Kubb <dkubb@fastmail.com>
 
 # Create system users
-RUN adduser -DS nginx
-RUN adduser -DS rails
+RUN adduser -DSH -G nogroup nginx \
```
Comment (mbj) [inline alpine-rails-nginx/Dockerfile:7]: Should we expand the short options here?

**Comment by mbj** [comment]
@dkubb Finished a code only pass.

File: alpine-rails-nginx/Dockerfile (hunk 1 above)
Comment (dkubb) [inline alpine-rails-nginx/Dockerfile:7]: There are no short options on Alpine Linux's `adduser`. Many of it's binaries are paired down and are not fully compatible with the common gnu alternatives.

**Pull request merged**